docs/about/features.md

```
@@ -50,37 +55,38 @@ We support:
+![AI Recommended Suggestions - CodeRabbit](/img/about/committable-suggestion.png)
 
-![Irony](/img/about/ironic-committable.png)
+![Irony in Committable Suggestions - CodeRabbit](/img/about/ironic-committable.png)
 
 ### 7. Learnings {#learnings}
 
-CodeRabbit learns from user chat interactions to gain tribal knowledge on your
-repositories and organization. CodeRabbit will apply and update these learnings
-in future reviews
+- **CodeRabbit** learns from user chat interactions to gain tribal knowledge on your
+repositories and organization. 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)